docs/API_walkthrough.md

```diff
@@ -0,0 +1,620 @@
+     "observedBy": {
+        "type": "Relationship",
+        "object": "urn:ngsi-ld:Sensor:02"
+     }
+  }
+}
+```
+
+* Since we updated both temperature and humidity, we can get the temporal evolution of those properties
+
+```shell
+http http://localhost:8080/ngsi-ld/v1/temporal/entities/urn:ngsi-ld:BeeHive:01 \
+    timerel==between \
+    time==2019-10-25T12:00:00Z \
+    endTime==2021-10-27T12:00:00Z \
+    Link:$CONTEXT_LINK
+```
+
+Sample payload returned showing the temporal evolution of temperature and humidity properties:
+
+```json
+{
+    "@context": [
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld"
+    ],
+    "humidity": [
+        {
+            "instanceId": "urn:ngsi-ld:Instance:a768ffb8-79e0-488f-9a4c-e7217ba2dff4",
+            "observedAt": "2019-10-26T21:35:52.986010Z",
+            "type": "Property",
+            "value": 58.0
+        },
+        {
+            "instanceId": "urn:ngsi-ld:Instance:28e44b9e-86f5-4bbb-a363-718d849d1782",
+            "observedAt": "2019-10-26T21:32:52.986010Z",
+            "type": "Property",
+            "value": 60.0
+        }
+    ],
+    "id": "urn:ngsi-ld:BeeHive:01",
+    "temperature": [
+        {
+            "instanceId": "urn:ngsi-ld:Instance:33642ff7-9b66-42c4-bcdf-9ca640cba782",
+            "observedAt": "2019-10-26T22:35:52.986010Z",
+            "type": "Property",
+            "value": 42.0
+        },
+        {
+            "instanceId": "urn:ngsi-ld:Instance:73226f04-1c47-49a6-ab88-976cb7493bea",
+            "observedAt": "2019-10-26T21:32:52.986010Z",
+            "type": "Property",
+            "value": 22.2
+        }
+    ],
+    "type": "BeeHive"
+}
+```
+
+* Let's update again the temperature property of the beehive:
+
+```shell
+http PATCH http://localhost:8080/ngsi-ld/v1/entities/urn:ngsi-ld:BeeHive:01/attrs \
+    Link:$CONTEXT_LINK < beehive_secondTemperatureUpdate.json
+```
+beehive_secondTemperatureUpdate.json
+
+```json
+{
+   "temperature": {
+     "type": "Property",
+     "value": 100,
+     "unitCode": "CEL",
+     "observedAt": "2020-05-10T10:20:30.98601Z",
+     "observedBy": {
+        "type": "Relationship",
+        "object": "urn:ngsi-ld:Sensor:01"
+     }
+   }
+}
+```
+
+* We can get the simplified temporal representation of the temperature property of the beehive:
+
+```shell
+http http://localhost:8080/ngsi-ld/v1/temporal/entities/urn:ngsi-ld:BeeHive:01?options=temporalValues \
+    attrs==temperature \ 
+    timerel==between \
+    time==2019-10-25T12:00:00Z \
+    endTime==2020-10-27T12:00:00Z \
+    Link:$CONTEXT_LINK
+```
+
+The sample payload returned showing the simplified temporal evolution of temperature and humidity properties:
+
+```json
+{
+    "@context": [
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld"
+    ],
+    "temperature": {
```

Comment:
not sure this is the payload you'll have

Reply:
updated